python/podio_gen/cpp_generator.py

```diff
@@ -505,12 +514,14 @@ def _write_all_collections_header(self):
     def _write_edm_def_file(self):
         """Write the edm definition to a compile time string"""
         model_encoder = DataModelJSONEncoder()
+        print(f"{self.datamodel_version=}")
```

Comment:
```suggestion
```
Debugging print? Otherwise it's going to be a bit weird like that whenever it appears with `self.`

Reply:
Ah, few seconds too late. This is indeed a debug print that I missed while having another look.